### PR TITLE
Enable find widget in goal panel

### PR DESCRIPTION
**Pull request by Alizter**
Fix https://github.com/ejgallego/coq-lsp/issues/305

Tested and works.

### CODE REVIEW

File: editor/code/src/goals.ts [hunk 1]
```diff
@@ -31,12 +31,14 @@ export class InfoPanel {
   }
 
   panelFactory() {
-    let webviewOpts: WebviewOptions = { enableScripts: true };
```
Comment (Alizter) [inline editor/code/src/goals.ts:34]: This was an intersection type so you couldn't add enableFindWidget here directly. I've inlined it but the type is really `WebviewOptions & WebviewPanelOptions`.

Comment (ejgallego) [reply editor/code/src/goals.ts:34]: You can also just remove the annotation (or add the intersection to the type annotation)

**Review by ejgallego — APPROVED** [review]
Oh, well spotted! Thanks a lot.

**Comment by ejgallego** [comment]
There is a prettier failure, I will rebase, uninline, and add a changes entry.

**Pull request merged**